recipes-kernel/linux/4.19/defconfigs/xenclient-stubdomain/defconfig

```diff
@@ -475,9 +481,7 @@ CONFIG_DMI_SCAN_MACHINE_NON_EFI_FALLBACK=y
 # Tegra firmware driver
 #
 CONFIG_HAVE_KVM=y
```

Comment:
Is it possible to drop this one since we have no need for KVM in the stubdom?

Reply:
Hi, @crogers1 .  `CONFIG_HAVE_*` variables indicate the availability of features but not selection.  X86 has KVM support, so it selects `CONFIG_HAVE_KVM`.  CONFIG_KVM controls building KVM.

CONFIG_KVM isn't in the defconfig since it's under CONFIG_VIRTUALIZATION which is disabled:
```
# CONFIG_VIRTUALIZATION is not set
```
```
grep KVM recipes-kernel/linux/4.19/defconfigs/xenclient-stubdomain/
# CONFIG_KVM_GUEST is not set
CONFIG_HAVE_KVM=y
```

Reply:
Never noticed that distinction.  Thanks for the info!